config/logging.yaml

```diff
@@ -49,15 +59,34 @@
 #     [%X{session}] session ID from cookies, else the request-ID
 #     [%X{session_safe}] random ID assigned per session, not available early on during
 #                        request logging, but prevents disclosure of the session ID
-#     [%X{string}] variable set using ::Logging.mdc['string'] =
-
+#     [%X{string}] variable set via MDC (::Logging.mdc['string'])
+#     [%x{string}] variable set via NDC (::Logging.ndc['string'])
+#
+# :sys_pattern:
+#   Logger line pattern for system log/journal. See :pattern for more info.
+#
+# :facility:
+#   System log/journal facility to use. One of LOG_AUTH, LOG_AUTHPRIV, LOG_CONSOLE,
+#   LOG_CRON, LOG_DAEMON, LOG_FTP, LOG_KERN, LOG_LPR, LOG_MAIL, LOG_NEWS, LOG_NTP,
+#   LOG_SECURITY, LOG_SYSLOG, LOG_USER, LOG_UUCP, LOG_LOCAL0 to LOG_LOCAL7.
+#
 :default:
   :colorize: false
   :console_inline: false
   :log_trace: false
   :level: info
-  :type: file
-  :pattern: "%d %.8X{request} [%c] [%.1l] %m\n"
+  :type: journald
+  :layout: multiline_pattern
+  :items:
+    - logger
+    - timestamp
+    - level
+    - message
+    - mdc
+    - ndc
+  :pattern: "%d [%.1l|%.3c|%.5X{request}] %m\n"
+  :sys_pattern: "[%.1l|%.3c|%.5X{request}] %m\n"
```

Comment:
Option `sys_pattern` is only used for syslog/journal, note there is no date prefix as it's not needed in this case.

Reply:
Good touch!

Reply:
Is this really used though? I have `layout: multiline_pattern` and `type: journald` - but my logs look like:

```
 3月 13 10:18:35 centos7-devel.lobatolan.home foreman[17690]: 2018-03-13T10:18:35 [I|app|a08c8]   Rendered /home/vagrant/foreman-plugin/redhat-access/app/views/dashboard/_insights_issues_widget.html.erb (1732.3ms)
 3月 13 10:18:35 centos7-devel.lobatolan.home foreman[17690]: 2018-03-13T10:18:35 [I|app|a08c8] Completed 200 OK in 3329ms (Views: 1727.8ms | ActiveRecord: 1558.9ms)
                                                                
```

So the date is printed twice.